data/rulesets.js

```diff
@@ -372,15 +372,38 @@ exports.BattleFormats = {
 		},
 		onValidateTeam: function (team, format) {
 			let abilityTable = {};
+			let base = {
+				airlock: 'cloudnine',
+				battlearmor: 'shellarmor',
+				clearbody: 'whitesmoke',
+				dazzling: 'queenlymajesty',
+				emergencyexit: 'wimpout',
+				filter: 'solidrock',
+				gooey: 'tanglinghair',
+				insomnia: 'vitalspirit',
+				ironbarbs: 'roughskin',
+				minus: 'plus',
+				powerofalchemy: 'receiver',
+				teravolt: 'moldbreaker',
+				turboblaze: 'moldbreaker',
+			};
 			for (let i = 0; i < team.length; i++) {
 				let ability = toId(team[i].ability);
 				if (!ability) continue;
-				if (ability in abilityTable) {
-					if (abilityTable[ability] >= 2) {
-						return ["You are limited to two of each ability by the Ability Clause.", "(You have more than two " + this.getAbility(ability).name + ")"];
+				if (ability in abilityTable || base[ability] in abilityTable) {
```

Comment:
You can remove all the duplicate `base[ability]` stuff by just doing `if (ability in base) ability = base[ability];` once, btw.